src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -0,0 +1,206 @@
+    """
+    config = loadConfig(config_filename)
+    try:
+      self.assertTlsHandshakeFailure(client_cert=config['sasCert'],
+                                     client_key=config['sasKey'])
+    except AssertionError as e:
+      self.SasReset()
+      
+      # Initiate Full Activity Dump
+      self._sas_admin.TriggerFullActivityDump()
+      try:
+         self._sas.GetFullActivityDump()
+      except AssertionError as e:
+         # Check if HTTP status is 403
+         self.assertEqual(e.args[0], 403)
+
+  def generate_SSS_12_default_config(self, filename):
+    """Generates the WinnForum configuration for SSS.12"""
+    # Create the actual config for domain proxy cert/key path
+
+    config = {
+        'sasCert': self.getCertFilename("sas_expired.cert"),
+        'sasKey': self.getCertFilename("sas_expired.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SSS_12_default_config)
+  def test_WINNF_FT_S_SSS_12(self,config_filename):
+    """Expired certificate presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    config = loadConfig(config_filename)
+    self.assertTlsHandshakeFailure(client_cert=config['sasCert'],
+                                   client_key=config['sasKey'])
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SSS_13(self):
+    """ Disallowed TLS method attempted during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    self.assertTlsHandshakeFailure(SAS_CERT, SAS_KEY, ssl_method=SSL.TLSv1_1_METHOD)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SSS_14(self):
+    """Invalid ciphersuite presented during registration.
+
+    Checks that SAS UUT response with fatal alert message.
+    """
+    self.assertTlsHandshakeFailure(SAS_CERT, SAS_KEY, ciphers='ECDHE-RSA-AES256-GCM-SHA384')
+
+  def generate_SSS_15_default_config(self, filename):
+    """ Generates the WinnForum configuration for SSS.15 """
+    # Create the actual config for domain proxy cert/key path
+
+    config = {
+        'sasCert': self.getCertFilename("sas_inapplicable.cert"),
+        'sasKey': self.getCertFilename("sas.key"),
+        'sasThUrl':'localhost:9001'
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SSS_15_default_config)
+  def test_WINNF_FT_S_SSS_15(self,config_filename):
+    """Certificate with inapplicable fields presented during registration.
+
+    Checks that SAS UUT response with tls connection succedded and response should be 104.
+    """
+    config = loadConfig(config_filename)
+
+    # Load the keys/certs and check that TLS handshake is valid
+    sas_cert = config['sasCert']
+    sas_key = config['sasKey']
+    self.assertTlsHandshakeSucceed(self._sas_admin._base_url, ['AES128-GCM-SHA256'], sas_cert, sas_key)
+    
+    # Initiate Full Activity Dump
+    self._sas_admin.TriggerFullActivityDump() 
+    try:
+      self._sas.GetFullActivityDump()
```

Comment:
idem as SCS_10: missing `ssl_cert` and `ssl_key`parameters

Reply:
The test cases SSS.10 and SSS.15 have been removed. These test cases have dependency on another PR#296.